iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/FileImplTest.java

```diff
@@ -37,73 +36,73 @@ void shouldParseEmptyFile() {
     File file = parseFile("");
     assertThat(file.getKind()).isEqualTo(DockerTree.Kind.FILE);
     assertTextRange(file.textRange()).hasRange(1,0,1,0);
```

Comment:
If we plan to keep this Body element, I think we should move most of those test into a new `BodyImplTest` element and only have test related to the `File` element itself here.